HerPublicWebsite/Controllers/QuestionnaireController.cs

```diff
@@ -810,4 +812,25 @@ public PathByActionArguments(string action, string controller, RouteValueDiction
             Values = values;
         }
     }
+    
+    private static string GetLocalAuthorityPendingMessagePartialViewPath(
+        Questionnaire questionnaire)
+    {
+        var partialViewName = questionnaire.CustodianCode switch
+        {
+            "2610" => "Broadland",
+            _ => "Default"
+        };
+        return $"~/Views/Partials/PendingLAMessage/${partialViewName}.cshtml";

```

Comment:
⚠️ You have not updated these paths to match the new directory structure of the partial views.